README.md

```diff
@@ -36,28 +36,29 @@ The `manager-api` is used to provide APIs for Dashboard, just like a bridge betw
 ```sh
 # e.g Initialization for MySQL
 $ mysql –uroot –p123456
-> source apisix-dashboard/api/script/db/schema.sql
+> source $PROJECT_ROOT_FOLDER/api/script/db/schema.sql
 ```
 
 2. Start the Apache APISIX.
 
 [Please follow this guide](https://github.com/apache/apisix#configure-and-installation)
 
-3. Set environment variables
+3. Check environment variables
 
-According to your local deployment environment, modify the environment variables in `$PROJECT_ROOT_FOLDER/api/run/run.sh`
+According to your local deployment environment, check the environment variables in `$PROJECT_ROOT_FOLDER/api/run/run.sh`, modify the environment variables if necessary.
 
 4. Build
 
 ```sh
-$ go build -o manager-api apisix-dashboard/api
+## make sure you are in the path of $PROJECT_ROOT_FOLDER
+$ go build -o manager-api ./api
 ```
 
 5. Run
 
 ```sh
-$ chmod +x apisix-dashboard/api/run/run.sh
-$ sh apisix-dashboard/api/run/run.sh
+## make sure you are in the path of $PROJECT_ROOT_FOLDER
```

Comment:
same issue here

Reply:
this is relation path, when run manager-api locally, need to run `run.sh` in the root path of apisix-dashboard